.github/PULL_REQUEST_TEMPLATE/new_probabilities_estimator.md

```diff
@@ -0,0 +1,40 @@
+## Contributor guidelines
+
+- [ ] I verify that I have read the
+    [devdocs](https://juliadynamics.github.io/Entropies.jl/dev/devdocs/).
+
```

Comment:
Similarly to the above, for me this sounds a bit too formal, and perhaps even intimidating.